nuget/RProvider.nuspec

```diff
@@ -30,8 +34,8 @@
     <file src="..\bin\RProvider.pdb" target="lib\net40" />
     <file src="..\bin\RProvider.fsx" target="." />
     <file src="..\nuget\RProvider.Runtime.dll.config" target="lib\net40" />
-    <file src="..\nuget\FSharp.Core.dll" target="lib\net40"/>
-    <file src="..\nuget\FSharp.Core.optdata" target="lib\net40"/>
-    <file src="..\nuget\FSharp.Core.sigdata" target="lib\net40"/>
+    <file src="..\tools\FSharp.Core.dll" target="lib\net40"/> 
```

Comment:
why don't you use FSharp.Core nuget package like https://github.com/fsprojects/Paket/blob/master/paket.dependencies#L11 ?